pkg/server/handler/appHandlers.go

```diff
@@ -26,6 +33,21 @@ func GetApps(c *gin.Context) {
 }
 
 func ListApps(c *gin.Context) {
+	kubeClient := c.MustGet("KubeClient")
+	envName := c.Param("envName")
+	envMeta, err := oam.GetEnvByName(envName)
```

Comment:
so looks like the envMeta is stored on the local file system?

Reply:
Fixed.